### ISSUE TITLE
Kops should add labels to nodes

### ISSUE THREAD

**Issue by shrinandj** [kubernetes/kops#742]
For a K8S cluster created using Kops, the master node is assigned an appropriate label:

`kubernetes.io/role=master`

However, the minion nodes do not have this label. It'll be useful to have this label on the minion nodes. Particularly for applying a nodeSelector to run daemonsets.

**Comment by tsupertramp** [comment]
Should be implemented as it is really useful but for anyone waiting on this: You can easily add
this label through ```kops edit ig nodes```
```
spec:
  nodeLabels:
    kubernetes.io/role: node
```

After a rolling update you have the appropriate label attached to your instances.

**Comment by mayank1029** [comment]
How we can automate these steps instead of doing it manually.

**Comment by chrislovecnm** [comment]
@mayank1029 use `kops create -f mycluster.yaml` rather than a kops edit.  You can access the full yaml with `kops get cluster -o yaml --full` and `kops get ig $CLUSTER_NAME -o yaml`.  You need to combine those two files into one single yaml file.  From there you can run your cluster off of yaml, rather than cli switches.

**Comment by chrislovecnm** [comment]
@shrinandj can we close this?

**Comment by lucascollino** [comment]
Today I found myself in need of doing something similar, but with Annotations instead of Labels.

Haven't seen the code in the repo nor the docs to do so.

Should I file a new issue?.
